### DISCUSSION TITLE
Egg Pterodactyl

### DISCUSSION THREAD

**Discussion by HakyuryITG** [datarhei/restreamer#601] [Ideas]
A very cool thing. Is it possible to install in the form of egg via Pterodactyl?

**Comment by svenerbeck** [comment]
Hello HakyuryITG,

Thank you for the compliment and your interest in our product!

Regarding your question about installing it as an "egg" via Pterodactyl: We haven't tested this specific use case and currently do not have resources allocated. However, we're always keen to learn from our community.

Have you tried doing it on your end? If you have or decide to give it a go, we would greatly appreciate any feedback or updates. It could benefit both us and other users in the community who might be looking into similar implementations.

Thank you once again, and we look forward to hearing from you!

Cheerio,
Sven
